app/indexers/indexes_metadata.rb

```diff
@@ -6,17 +6,16 @@ def generate_solr_document
       solr_doc['title_ssi'] = object.title.first
       solr_doc['date_created_ssi'] = object.date_created.first
       solr_doc['creator_tesim'] = creator_alternate_names(object).to_a + creator_names(object).to_a
```

Comment:
Do we need to worry about blank creator or alternate names here too?

Reply:
haven't run into problems from it, but put the blank rejection code into both methods in case.